backend/calculateDistance.js

```diff
@@ -0,0 +1,13 @@
+const calculateDistance = (userMoodScores, songMoodScores) => {
+  let distance = 0;
+
+  Object.keys(userMoodScores).forEach((mood) => {
+    const userScore = userMoodScores[mood] || 0;
```

Comment:
I think doing the solution for the songscore is a great solution to the problem. I think that for the userScores you could potentially divide the number by 5 in order to normalize it, and this would help there be some more specificity with the recommendations.